soccer/gameplay/robocup-py.cpp

```diff
@@ -94,6 +95,11 @@ void OurRobot_move_to_direct(OurRobot* self, Geometry2d::Point* to) {
     if (to == nullptr) throw NullArgumentException("to");
     self->moveDirect(*to);
 }
+void OurRobot_move_to_direct_end_vel(OurRobot* self, Geometry2d::Point* to,
```

Comment:
Yeah the moveDirect is sort of a hack.  The reason it normally avoids the ball is because we add a large circular obstacle around it whenever we're in the STOPPED state.  See this method: https://github.com/RoboJackets/robocup-software/blob/master/soccer/Robot.cpp#L407